lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1623,6 +1631,7 @@ def cmd_sessions(*args)
             if session.respond_to?(:response_timeout)
```

Comment:
I can't comment on the line of code above, but looks like this is missing the timeout message in the `cmd` scenario


https://github.com/rapid7/metasploit-framework/blob/cd0ccd1716cd5ae71be33c618dd613268932f0cf/lib/msf/ui/console/command_dispatcher/core.rb#L1584-L1588C1